package.json

```diff
@@ -96,8 +96,8 @@
     "depcheck/@babel/parser": "7.16.4"
   },
   "dependencies": {
-    "cross-fetch": "^3.1.5",
-    "nanoid": "^3.3.4"
+    "nanoid": "^3.3.4",
+    "node-fetch": "2"
```

Comment:
We cannot use 'require' with node-fetch 3 (since it is a full ESM module). Node-fetch 2 is still widely supported and new versions are still coming up.